### ISSUE TITLE
I'am new here and please help me to apply WordPress.

### ISSUE THREAD

**Issue by daron4ever** [GoogleCloudPlatform/php-docs-samples#287]
@tmatsuo  @jmdobry   
The media file was saved localhost:8080/_ah/gcs/app_default_bucket/filename
It worked during the development time.
But when I continuously developed next day, the file in the Media library has been missing.

I followed the google [documentation ](https://cloud.google.com/storage/docs/reference/libraries#client-libraries-install-php) to solve the above issue.

So, I have installed the client library using `$ composer require google/cloud` /PATH/TO/MY_PROJECT/ directory where app.yaml is being.
And, I have this script linked from Google. (php-docs-samples/storage/quickstart/quickstart.php)

I don't know how to run this client libarary with my WordPress project.

The help would be appreciated.

**Comment by bshaffer** [comment]
Can you explain what issue you're having exactly? It sounds like you are uploading a file locally and you do not see it in your production instance. Check your bucket in the [Google Cloud Console](https://console.cloud.google.com/storage) and ensure the file exits there. Also be sure you're using the same bucket locally and in production.

**Comment by daron4ever** [comment]
I have found the solution.

**Comment by jgacuca567** [comment]
@daron4ever please post your solution

**Comment by daron4ever** [comment]
I installed the 'Google App Engine for WordPress' plugin and it used  "/_ah/gcs/app_default_bucket/" at local development for Media files.
I realized the "/_ah/gcs/app_default_bucket/" didn't save any Media files, so, I deactivated the plugin and it led to use the wp-content>uploads default directory for Media files at local development environment. So I just uploaded the Media files to the Cloud Storage when I deploy the app.
